docs/community/index.md

```diff
@@ -5,13 +5,32 @@ There are several different ways to be a part of the napari community. From
 being a code or documentation contributor, to creating educational content or
 examples, you are welcome to help develop and improve napari.
 
+## Questions
+
 To ask questions and interact with the team, you can join our
 [zulip chat](https://napari.zulipchat.com/login/) or participate in our
 [meetings](meeting-schedule).
 
+To ask general question about napari usage to whole community of napari user, also for plugins 
+You may visit ![forum image.sc logo](../images/image_sc_logo.png) [forum.image.sc/tag/napari](https://forum.image.sc/tag/napari)
+
+## News and updates
+
+To trace latest news from napari you may follow us on:
+ * ![twitter logo](../images/twitter_logo.svg) twitter/X [napari_imaging](https://x.com/napari_imaging/) 
+ * ![mastodon logo](../images/mastodon_logo.svg) mastodon [fosstodon.org/@napari](https://fosstodon.org/@napari)    
+
+
+## Contributing and Bug reporting
+
+The napari development happens in napari repository [github.com/napari/napari](https://github.com/napari/napari). 
+If you spot a bug, please check our [issues list](https://github.com/napari/napari/issues) and if you 
+do not found matching report pleas open new issue. 
+
 If you are interested in contributing, check out our
-[contributing guide](contributing).
+[contributing guide](napari-contributin).
```

Comment:
```suggestion
[contributing guide](napari-contributing).
```

This is why the docs build is failing, see https://app.circleci.com/pipelines/github/Czaki/napari-docs/110/workflows/1a59d4a2-ea8b-46ea-8d92-b8911748d94e/jobs/110?invite=true#step-106-128362_126